mw-webapp/src/App.tsx

```diff
@@ -1,16 +1,58 @@
-import {useState} from "react";
+import {useEffect, useState} from "react";
 import {RouterProvider} from "react-router-dom";
+import {Modal} from "src/component/modal/Modal";
+import {Text} from "src/component/text/Text";
+import {HeadingLevel, Title} from "src/component/title/Title";
+import {VerticalContainer} from "src/component/verticalContainer/VerticalContainer";
+import {HealthCheckDAL} from "src/dataAccessLogic/HealthCheckDAL";
 import {
   DEFAULT_NOTIFICATION_SETTINGS,
   globalContext,
 } from "src/GlobalContext";
+import {languageStore} from "src/globalStore/LanguageStore";
 import {router} from "src/router/Router";
+import {LanguageService} from "src/service/LanguageService";
+import styles from "src/App.module.scss";
 
 /**
  * App
  */
 export const App = () => {
   const [isInitialized, setIsInitialized] = useState(false);
+  const [isApiWorking, setIsApiWorking] = useState(true);
+
+  const {language} = languageStore;
+
+  /**
+   * Check health of the API
+   */
+  const checkApiHealth = async () => {
+    const {isWorkingApi} = await HealthCheckDAL.checkApiHealth();
+    setIsApiWorking(isWorkingApi);
+  };
+
+  useEffect(() => {
+    checkApiHealth();
+  }, []);
+
+  if (!isApiWorking) {
+    return (
+      <Modal
+        isOpen={!isApiWorking}
+        trigger={<div className={styles.modalContent} />}
```

Comment:
is it possible to use for trigger empty fragment <> </> instead of div with classname

Reply:
fixed